bin/build-pouchdb.js

```diff
@@ -208,8 +202,6 @@ function doBuildAll() {
 function doBuild() {
   if (process.env.BUILD_NODE) { // rebuild before "npm test"
     return doBuildNode();
-  } else if (DEV_MODE) { // rebuild during "npm run dev"
-    return doBuildDev();
```

Comment:
I can't see any reason why we shouldn't just do the full build; during debugging we still need to debug files like `dist/pouchdb.find.js` and `dist/pouchdb.memory.js`.